components/content/Text/Text.css

```diff
@@ -0,0 +1,188 @@
+/* Font */
+
+.diamond-text-font-body {
```

Comment:
We should set the font-family in here, it's common to have a different family for body and heading.